website/src/pages/menu-button.mdx

```diff
@@ -133,6 +134,13 @@ function Example() {
 }
 ```
 
+##### Menu `keepScrollPosition`
+
+`keepScrollPosition: boolean`
+
+When a menu is open the focus is trasferred to the menu itself. In the case the menu list element height is larger than the window height this focus will scroll the page making it jumpy.
+Passing `true` on this prop will still focus on the menu element but keeping the position of the scroll on the page.
```

Comment:
A few grammatical edits and spell checks:

```
When a `<Menu>` is opened the focus is transferred to the `<MenuList>`. If the `<MenuList>` height is longer than the viewport height, the focus will scroll the page so that the top of the `<MenuList>` is flush with the top of the viewport, which may result in a visual "jump".
Passing `true` for this prop will still focus the `<MenuList>`, but avoid the scroll position change, regardless of the `<MenuList>` height.
```